### PR TITLE
 Improved manual testing with utils.sh, titles, and balance tables

### PR DESCRIPTION
**Pull request by MattPoblete**
This pull request improves the manual testing process by making the following changes:

- A new file, utils.sh, is added to the manual_testing directory. This file contains functions for displaying relevant information, such as test results and token balances.
- Titles are added to each test to identify them at the beginning.
- Tables are added to the end of each test to show the balances of the tokens.

### CODE REVIEW

**Comment by esteblock** [comment]
Hi @MattPoblete Please fix
- Change name of column to "diff after add_liquidity", or "diff after swap"
- Difference should be positive if balance after is greater.
- Add balance of LP token

File: README.md [hunk 1]
```diff
@@ -220,5 +220,42 @@ To run all the transsactions we could do on soroswap protocol just run (inside s
...
+The first step is to start the development environment. This can be done by executing the following command:
+
+
+```
+bash scripts/quickstart.sh standalone
+
+```
+
+This command will start a standalone development environment with the contracts running.
+
```
Comment (chopan123) [inline README.md:240]: Deploy tokens n pairs missing

File: scripts/manual_testing/refactor/add_liquidity.ts [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    const rpcUri1 = "http://stellar"
+    const routerContractAddress = "CCCGECJFVPS4KDZUEXOSFKWEVX3QKDYATMWYSQ5WU6TOGYHHN6DMVYLT"
+    const txBuilder = new TransactionBuilder(
+        `${rpcUri1}:8000`,
+        `${rpcUri1}:8000/soroban/rpc`,
+        `${rpcUri1}:8000/friendbot?addr=`,
+        routerContractAddress,
+        network1
+    ); */
+export const add_liquidity = async (txMaker:TransactionBuilder) => {
```
Comment (chopan123) [inline scripts/manual_testing/refactor/add_liquidity.ts:15]: use camelCase

Add docs

File: scripts/manual_testing/refactor/all.ts [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+        break;
+    default:
+        console.log("Usage: local|public")
+        console.log("local: use contracts from the .soroban folder (local deployements)")
+        console.log("public: use contracts from the /public folder (addresses in production?)")
+        break;
+}
+
+const testAll = async () => {
+    let apiUri = "http://soroswapCoreApi:8010"
```
Comment (chopan123) [inline scripts/manual_testing/refactor/all.ts:43]: handle when testnet or standalone

File: scripts/manual_testing/refactor/mint.ts [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+const routerContractAddress = "CDPOUV6Q2DLFOQZ3TA5ACUADUWB56HCQAAJWLVV27INLWC7MVAJXY3EU"
+const txBuilder = new TransactionBuilder(
+    `${rpcUri1}:8000`,
+    `${rpcUri1}:8000/soroban/rpc`,
+    `${rpcUri1}:8000/friendbot?addr=`,
+    routerContractAddress,
+    network1
+); */
+
+export const mint = async (txMaker: TransactionBuilder) => {
```
Comment (chopan123) [inline scripts/manual_testing/refactor/mint.ts:17]: add docs

File: scripts/manual_testing/refactor/remove_liquidity.ts [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+    const rpcUri1 = "http://stellar"
+    const routerContractAddress = "CDPOUV6Q2DLFOQZ3TA5ACUADUWB56HCQAAJWLVV27INLWC7MVAJXY3EU"
+    const txBuilder = new TransactionBuilder(
+        `${rpcUri1}:8000`,
+        `${rpcUri1}:8000/soroban/rpc`,
+        `${rpcUri1}:8000/friendbot?addr=`,
+        routerContractAddress,
+        network1
+    ); */
+export const remove_liquidity = async (txMaker:TransactionBuilder) => {
```
Comment (chopan123) [inline scripts/manual_testing/refactor/remove_liquidity.ts:15]: useCamelCase 
add docs

File: scripts/manual_testing/refactor/utils/utils.ts [hunk 6]
```diff
@@ -0,0 +1,256 @@
...
+  // Create the directory if it doesn't exist
+  if (!fs.existsSync(dirPath)) {
+    fs.mkdirSync(dirPath, { recursive: true });
+  }
+
+  const data = JSON.stringify(accounts, null, 2);
+  fs.writeFileSync(filepath, data);
+}
+
+export function saveContracts(tokens: tokenContract[]): void {
```
Comment (chopan123) [inline scripts/manual_testing/refactor/utils/utils.ts:46]: Add docs

File: scripts/manual_testing/utils.sh
Comment (chopan123) [inline scripts/manual_testing/utils.sh:1]: write docs before every function

File: README.md [hunk 7]
```diff
@@ -220,5 +220,52 @@ To run all the transsactions we could do on soroswap protocol just run (inside s
...
+bash scripts/quickstart.sh standalone
+```
+2.  Start the development container. To start the development container, in a new terminal, run the command:
+ ```
+bash scripts/run.sh
+```
+
+3.  Deploy the tokens. Once the development container is started, run the command:
+```
+bash scripts/deploy_tokens_n_pairs"
```
Comment (chopan123) [inline README.md:244]: that will fail

**Review by chopan123 — COMMENTED** [review]
Add functionality for local and public

File: scripts/manual_testing/refactor/all.ts [hunk 8]
```diff
@@ -0,0 +1,88 @@
...
+            console.log(colors.green, 'Using testnet network')
+            apiUri = "https://horizon-testnet.stellar.org"
+            rpcUri = "http://stellar"
+            routerContractAddress = await getRouterContractAddress(apiUri, network)
+            txMaker = new TransactionBuilder(
+                `${rpcUri}`,
+                `${rpcUri}/soroban/rpc`,
+                "https://friendbot.stellar.org/?addr=",
+                routerContractAddress,
+                network
```
Comment (chopan123) [inline scripts/manual_testing/refactor/all.ts:58]: You have duplicated code, 

move txMaker to after the switch statement (define variables inside switch, define txMaker only once after the switch)

**Review by chopan123 — COMMENTED** [review]
add local || public functionality

File: scripts/manual_testing/refactor/all.ts (hunk 8 above)
Comment (MattPoblete) [reply scripts/manual_testing/refactor/all.ts]: Fixed!

**Pull request merged**